lib/tasks/data.rake

```diff
@@ -26,4 +26,16 @@ namespace :data do
       end
     end
   end
+
+  desc 'Migrate working pattern to array enum'
+  namespace :working_pattern do
+    task migrate: :environment do
+      Vacancy.where(working_patterns: nil)
+             .where.not(working_pattern: nil)
+             .each do |vacancy|
+        vacancy.working_patterns = [vacancy.working_pattern]
```

Comment:
Do we need some tests to ensure we're not using the old `working_pattern` anywhere if it's still set?

Reply:
I think we have a potential race condition where we need to migrate the existing working pattern to the new column, then remove the old one completely in a separate PR

Reply:
Yeah, I guessed that was why the old field was still there. There will be an intermediary time when we have both fields, however, and during that time, it's plausible that we could be reading or writing to the old field when we mean to hit the new one. I'm not sure where's best to make sure that isn't happening. My instict was to check now, since if we have accidental writes, that would require the data migration to be run again after fixing whatever issue was there, but happy to be overruled :).

Reply:
Yeah I think you're right. I've done a search for anywhere where we might be using the old one and can't see anything. We can have some extra confidence from the level of coverage our tests have but maybe it needs something else?